libs/application/templates/official-journal-of-iceland/src/screens/InvolvedPartyScreen.tsx

```
@@ -0,0 +1,107 @@
+  setSubmitButtonDisabled,
+}: OJOIFieldBaseProps) => {
+  const { updateApplication } = useApplication({
+    applicationId: application.id,
+  })
+  const { formatMessage: f } = useLocale()
+  const { setValue } = useFormContext()
+
+  const { involvedParties, error, loading } = useInvolvedParties({
+    applicationId: application.id,
+    onError: () => {
+      setSubmitButtonDisabled && setSubmitButtonDisabled(true)
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)